qa/tasks/cephfs/test_quota.py

```diff
@@ -160,3 +160,37 @@ def test_disable_enable_human_readable_quota_values(self):
         self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "0M")
         self.assertEqual(self.mount_a.getfattr("./subdir",
                                                "ceph.quota.max_bytes"), None)
+        
+    def test_human_readable_float_quota_values(self):
+        """
+        test human-readable float values for setting ceph.quota.max_bytes
+        """
+        self.mount_a.run_shell(["mkdir", "subdir"])
+
+        self.assertEqual(self.mount_a.getfattr("./subdir",
+                                               "ceph.quota.max_bytes"), None)
+
+        readable_values = {"1.5K": "1536",
+                           "3.6Ki": "3686",
+                           "5.5Mi": "5767168",
+                           "2.5G": "2684354560",
+                           "2.2Ti": "2418925581107"}
+        for readable_value in readable_values:
+            self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes",
+                                  readable_value)
+            self.assertEqual(self.mount_a.getfattr(
+                "./subdir", "ceph.quota.max_bytes"),
+                readable_values.get(readable_value))
+
+    def test_human_readable_float_quota_invalid_values(self):
+        """
+        test float invalid values for ceph.quota.max_bytes
+        """
+
+        self.mount_a.run_shell(["mkdir", "subdir"])
+
+        invalid_values = ["21.A", "3.", "19.Ki"]
```

Comment:
Since you added two new value checks you should consider all three cases that could return err:
1. Byte validate (new)
2. Float validation (new)
3. strict_iec_cast

Currently you have:
1. '21.A' (Invalid Float)
2. '3.' (Invalid Byte)
3. '19.1Ki' (Invalid Float)

A potential additional check could be: 'setfattr -n ceph.quota.max_bytes -v 19.0A subdir/'. That would exercise the `strict_iec_cast` case.

Reply:
Yup, I agree. Thanks, I missed  that case:). Updated